doh.c

```diff
@@ -666,6 +666,8 @@ static int initprobe(struct dnsprobe *p, int dnstype, char *host,
     curl_easy_setopt(curl, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_2TLS);
     curl_easy_setopt(curl, CURLOPT_PRIVATE, p);
     curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
+    curl_easy_setopt(curl, CURLOPT_SSL_VERIFYPEER, insecure_mode);
```

Comment:
You probably also want to disable `CURLOPT_SSL_VERIFYHOST` in insecure mode so that the names in the cert don't have to match the hostname used in the URL...

Reply:
Good point. I'll update it.